todo.js

```diff
@@ -42,6 +59,60 @@ function createDeleteButton(todoId) {
   return deleteButtonElement;
 }
 
+function createEditButton(todoId) {
+  const editButtonElement = document.createElement("button");
+  editButtonElement.innerHTML = "Edit";
+  editButtonElement.addEventListener("click", function () {
+    handleTodoEdit(todoId);
+  });
+  return editButtonElement;
+}
+
+function createUpdateButton(todoId) {
+  const updateButtonElement = document.createElement("button");
+  updateButtonElement.innerHTML = "Update";
+  updateButtonElement.addEventListener("click", () => handleTodoUpdate(todoId));
+  return updateButtonElement;
+}
+
+function createCancelButton(todoId) {
+  const cancelButtonElement = document.createElement("button");
+  cancelButtonElement.innerHTML = "Cancel";
+  cancelButtonElement.addEventListener("click", () => cancelEdit(todoId));
+  return cancelButtonElement;
+}
+
+function handleTodoEdit(todoId) {
+  todos.forEach((todo) => {
+    if (todo.id === todoId) {
+      todo.editState = true;
+      return;
+    }
+    todo.editState = false;
+  });
```

Comment:
Can we utilize find() method as per last discussion in the daily meeting?

```suggestion
const todo = todos.find({ id } => id === todoId)

if(todo === undefined){
    return
}

todo.editState = true;
```